### PR TITLE
flake8 Integration

### PR DESCRIPTION
**Pull request by ydjain**
Added a few lines to tox.ini after reading [this](http://flake8.pycqa.org/en/latest/user/index.html) . Hope this works.

### CODE REVIEW

**Comment by ChristianKuehnel** [comment]
Hi @ydjain,

thx for the patch. How do you call flake8 from tox? In Home Assistant, flake8 is automtically called when running tox. When I run tox, I do not see any results from flake8 and I would expect to get *a lot* of warnings...

**Comment by ydjain** [comment]
Hi I have realised my stupidty :stuck_out_tongue_closed_eyes:  and have made changes accordingly...I have added pylint too...all issues raised by flake8 have been resolved

File: tox.ini [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py34, py35, py36, integration_tests
+envlist = py34, py35, py36,pylint,lint, integration_tests
```
Comment (ThomDietrich) [inline tox.ini]: spaces missing

Comment (ydjain) [reply tox.ini]: Corrected!

**Comment by ChristianKuehnel** [comment]
looks great, thx!

**Pull request merged**